### DISCUSSION TITLE
Found value 9 for 'DRS' in car_data

### DISCUSSION THREAD

**Discussion by TracingInsights** [theOehrly/Fast-F1#199] [Ideas]
Here's the value_counts() for DRS column for Canadian GP

0  -   508618
1   -  171270
8   -   64910
12  -   34752
14   -   6787
10   -    489
9   -     450
2    -    178
3     -     2

When updating the documentation, please include 9 too

**Comment by theOehrly** [comment]
Interesting, any chance that you have also figured out what this value stands for?
